.github/workflows/inactive_issues.yml

```diff
@@ -32,3 +33,24 @@ jobs:
           stale-pr-message: "This PR is stale because it has been open for 60 days with no activity."
 
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      # This step marks kind/enhancement and kind/bug as stale, after a longer period of time.
+      - uses: actions/stale@v9
+        with:
+          # Mark enhancement and bug issues stale after 6 months, then close them after another month of inactivity.
+          days-before-issue-stale: 180
+          days-before-issue-close: 30
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 180 days with no activity."
+          close-issue-message: "This issue was closed because it has been inactive for 30 days since being marked as stale."
```

Comment:
This is probably overkill, but I wonder if we should differentiate this set of messages from the one for normal (non-triaged) issues. This is for any user who might notice that some issues have a "This issue is stale because it has been open for 30 days with no activity." on one stale is and then see "This issue is stale because it has been open for 180 days with no activity." and then wonder why they had different time periods before being marked as stale.